gmail-google-calender-openaisdk-demo/app/globals.css

```
@@ -0,0 +1,237 @@
+  color-scheme: dark light;
+  --bg: #0b0c0e;
+  --surface: #131519;
+  --surface-2: #1a1d22;
+  --border: #262a31;
+  --text: #e7e9ec;
+  --muted: #868e9b;
+  --accent: #4c8dff;
+  --ok: #3fb950;
+  --off: #f2545b;
+  --radius: 10px;
+  --mono: ui-monospace, "SF Mono", "Cascadia Code", "JetBrains Mono", Menlo, monospace;
```

Stylelint (17.14.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)